src/components/Calls/CallNotification.js

```diff
@@ -11,50 +11,53 @@ import { createClassName, getAvatarUrl } from '../helpers';
 import styles from './styles.scss';
 
 
-export const CallNotification = ({ callProvider, callerUsername, url, dispatch } = { callProvider: undefined, callerUsername: undefined, dispatch: undefined }) => {
-	const [show, setShow] = useState(!!callProvider && !!callerUsername && !!dispatch && !!url);
+export const CallNotification = ({ callProvider, callerUsername, url, dispatch, time } = { callProvider: undefined, callerUsername: undefined, dispatch: undefined, time: undefined }) => {
+	const [show, setShow] = useState(true);
+	console.log('asd', show);
 
 	const acceptClick = async () => {
 		setShow(!{ show });
-
 		switch (callProvider) {
 			case constants.jitsiCallStartedMessageType: {
 				window.open(url);
-				await dispatch({ incomingCallAlert: null });
 				break;
 			}
 			case constants.webrtcCallStartedMessageType: {
 				// TODO: add webrtc code here
+				await dispatch({ ongoingCall: { callStatus: 'accept', time: { time } } });
 				break;
 			}
 		}
 	};
 
 	const declineClick = async () => {
 		await dispatch({ incomingCallAlert: null });
+		await dispatch({ ongoingCall: { callStatus: 'decline', time: { time } } });
 	};
 
 	return (
-		<div className={createClassName(styles, 'call-notification')}>
+		<div>
```

Comment:
I didn't understand this change
![image](https://user-images.githubusercontent.com/34130764/128010114-4bf42496-f89b-4dc9-85f4-b97ae3819f79.png)

I think you're introducing a new parent div, but I'm not sure what's the use of it. Could you please explain its usage incase I've missed something?

Reply:
The div with 'call-notification' should be present only when the show value is true, or else when the show value is false then also style with classname 'call-notification' will be applied and that we do not need. So, I replace the div with classname 'call-notification' with new div.